lib/tasks.dart

```diff
@@ -141,20 +170,18 @@ class Task {
 
   external set selfLink(String selfLink);
 
-
+  // figure out how to seamlessly convert these to DateTimes for dart
```

Comment:
```suggestion
  // Figure out how to seamlessly convert these to DateTimes for dart.
```

If the API returns a String, it's probably best to just go with it, and transform it in user-code (potentially providing a helper function).
At some point I tried to dartify the APIs, and it was just too much work. It's easier to live with the original API...

If the API returns a `Date` object, it should automatically be transformed to a `DateTime` (I believe).